src/Sarif.Sarifer/AnalyzeSolutionCommand.cs

```diff
@@ -69,7 +73,27 @@ private void MenuCommandCallback(object caller, EventArgs args)
                 targetFiles.AddRange(project.GetMemberFiles());
             }
 
-            this.backgroundAnalysisService.AnalyzeAsync(solution.FullName, targetFiles).FileAndForget(FileAndForgetEventName.BackgroundAnalysisFailure);
+            this.backgroundAnalysisService.AnalyzeAsync(solution.FullName, targetFiles, cancellationTokenSource.Token)
```

Comment:
>cancellationTokenSource [](start = 88, length = 23)

we should enforce a standard in this code of using this. prefix, yes?

Reply:
yes! just updated the sarifer project!

---
In reply to: [532104079](https://github.com/microsoft/sarif-visualstudio-extension/pull/291#discussion_r532104079) [](ancestors = 532104079)